administrator/components/com_associations/helpers/associations.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Associations component helper.
+ *
+ * @since  1.6
+ */
+class AssociationsHelper extends JHelperContent
+{
+	public static $extension = 'com_associations';
+
+	/**
+	 * Configure the Linkbar.
+	 *
+	 * @param   string  $vName  The name of the active view.
+	 *
+	 * @return  void
+	 *
+	 * @since   1.6
+	 */
+	public static function addSubmenu($vName)
+	{
+		JHtmlSidebar::addEntry(
+			JText::_('JGLOBAL_ARTICLES'),
+			'index.php?option=com_content&view=articles',
```

Comment:
It was for my initial idea to have the sidebar for each component, deprecated now.